src/Services/LDAPService.php

```diff
@@ -1102,7 +1102,7 @@ public function addLDAPUserToGroup($userDn, $groupDn)
      * Change a members password on the AD. Works with ActiveDirectory compatible services that saves the
      * password in the `unicodePwd` attribute.
      *
-     * @todo Use the Laminas\Ldap\Attribute::setPassword functionality to create a password in
+     * Use the Laminas\Ldap\Attribute::setPassword functionality to create a password in
```

Comment:
Leave this as a additional comment

Reply:
It seems like this is a todo comment saying we _should_ use that, which we don't currently. We should probably delete this comment and create an issue for it.